stack/__main__.py

```diff
@@ -80,7 +80,7 @@ def org_role_id(id_suffix: str) -> str:
     )
 
     # Enable Dataproc until the Hail Query Service is ready.
-    cloudidentity = gcp.projects.Service(
+    _ = gcp.projects.Service(
```

Comment:
Nice catch, but do we actually need the assignment at all?

Reply:
The `_ = ` is the python semantic for ignoring the return value. Otherwise some linters might complain you're discarding the result of a function.

Reply:
Right, I think in other places we don't use that pattern though. Fine either way! :)